src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder report = createReport(fromFileName);
+        writeToFile(report, toFileName);
+    }
+
+    private void writeToFile(StringBuilder report, String toFileName) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            writer.write(report.toString());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file" + e);
```

Comment:
Write a more informative message, and the same for the second exception
```suggestion
            throw new RuntimeException("Can't write data to the file " + toFileName, e);
```